### ISSUE TITLE
Change this repository + app name

### ISSUE THREAD

**Issue by Salamandar** [YunoHost-Apps/bitwarden_ynh#103]
This app is NOT bitwarden. It is bitwarden_rs, an unofficial fork of Bitwarden written in rust.

The name of the app is deceiving, please change it. I wanted to test the official software, not the fork that is not yet feature-complete !

**Comment by Go-rom** [comment]
Hello,

i think you're making a mistake. Here is the repository dedicated to the Bitwarden app for Yuhohost.

You'll find the right repository (bitwarden_rs) here : https://github.com/dani-garcia/bitwarden_rs

**Comment by yalh76** [comment]
I don't use bitwarden. I've packaged it because a lot of people wanted it... 

When I looked at bitwarden wiki, I found no option to build it, and the requirement are:
- .NET Core 3.1 SDK
- SQL Server 2017
- Docker

Far from what was available in YunoHost...

I found bitwarden_rs ... with their introduction phrase: 'his is a Bitwarden server API implementation written in Rust compatible with upstream Bitwarden clients*, perfect for self-hosted deployment where running the official resource-heavy service might not be ideal.'

So... I think we will never have official bitwarden available in YunoHost... I also think bitwarden_rs is a good choice for a bitwarden implementation for YunoHost. Just my opinion.

In conclusion:
1- I don't use Bitwarden, so if someone want to be the maintainer, feel free
2- I don't care of the name, so if someone want to change it, feel free to do the job ;)
